exercise_McLean.Rmd

```diff
@@ -0,0 +1,120 @@
+
+DONE! YEAH!
+
+4.  Complete the sequence of coding tasks below (to the best of your ability).
+
+Okay
+
+5.  Save, Add, Commit, and Push your changes to this file.
+
+Can do!
+
+6.  Go to **your** forked Week-1-Intro Repo, click the 'Contribute' drop down at the top and click the 'Open Pull Request' button (if this is grayed out that means you have not pushed any changes to GitHub yet).
+
+7.  Provide a title, short description/notes of your changes, and most importantly under 'Reviewers' on the right, type in the username of your **assigned PR reviewer** (which you can find in the 'PR_reviewer_assignments.md' document in this repository. When you are all set, submit the pull request.
+
+8.  **Review your assigned pull request**. You should receive a notification either via GitHub (you can check by clicking the 'notifications' icon directly next to your profile photo in the upper right corner), or you will receive an email link to the email you registered your GitHub account to.
+
+    Follow [these guidelines for reviewing a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request), give some meaningful feedback, and when you're done 'Approve' the pull request (do NOT close or merge).
+
+## The coding exercise:
+
+Work through the steps below to complete this exercise (to the best of your ability) and then save, add, commit and push this file with your coding edits to your forked repo and follow the instructions above for completing the assignment via creating and reviewing a pull request.
+
+Load in the tidyverse (only package(s) needed for this exercise:
+
+```{r}
+library(tidyverse)
+```
+
+Read in the 'kenya_county_climates.csv' in the 'data/' folder and assign it to an environmental variable (Hint: use 'read_csv()').
+
+```{r}
+env <- read_csv("C:/Users/Admin/Desktop/ESS523A/Week-1-Intro/data/kenya_county_climates.csv")
+```
+
+Write a chain of operations that does the following:
+
+1.  `select()` just the columns that have "Precipitation" in the name (so we are only interested in the precipitation data), AND the 'County' column.
+
+
+```{r}
+CountyPrecip <- select(env, County, contains("Precipitation"))
+```
+
+
+
+2.  `filter()` all the rows where 'Precipitation_585_90' is **greater than** the mean of all county precipitation values.
+
+```{r}
+FilteredPrecip <- CountyPrecip %>%
+  filter(Precipitation_585_90 > mean(Precipitation_585_90))
+  
+```
+
+
+3.  `pivot_longer()` all the precipitation columns (so everything *except* 'County'), so we have one column for all precip values and one column with the value's associated variables.
+
+```{r}
+
+Pivot <- pivot_longer(CountyPrecip, contains("Precipitation"))
+
+
+```
+
+Question: How many counties have predicted precip values greater than the total average for all counties?
+
+```{r}
+CountyCount <- Pivot %>%
+  group_by(County, value)
```

Comment:
Great job! I forgot you could use the group_by function here, that would've been useful in my code.